client/tailwind.config.js

```diff
@@ -49,6 +52,8 @@ module.exports = {
       inherit: 'inherit',
       current: 'currentColor',
       transparent: 'transparent',
```

Comment:
It might be good to add a bit of clarity here with a comment.

```suggestion
      transparent: 'transparent',
      /* allow system colours https://www.w3.org/TR/css-color-4/#css-system-colors */
```